Asset/data/asset/predicates/mob/0056.thunder_trifler/golden_sword.json

```
@@ -0,0 +1,13 @@
+{
+    "condition": "minecraft:entity_properties",
+    "entity": "this",
+    "predicate": {
+        "equipment": {
+            "mainhand": {
+                "items": [
+                    "minecraft:golden_sword"
+                ]
```

```suggestion
                "item":
                    "minecraft:golden_sword"
```
なんか動いてなかったよ
itemsじゃなくてitemに変えて[]を消したら動いた